config.json

```diff
@@ -1,21 +1,44 @@
 {
     "client": {
-        "sciper": "000000",
-        "train_name": "Player",
         "host": "127.0.0.1",
         "port": 5555,
         "screen_width": 500,
         "screen_height": 360,
-        "cell_size": 20,
         "leaderboard_width": 280,
         "manual_spawn": false,
-        "control_mode": "manual"
+        "control_mode": "manual",
+        "server_timeout_seconds": 2.0,
+        "game_mode": "competitive",
+
+        "competitive_agent": {
+            "sciper": "000000",
+            "name": "Player",
+            "path": "agents.agent"
+        },
+        "local_agents": [
+            {
+                "name": "AgentExample1",
+                "path": "agents.agent_example1"
+            },
+            {
+                "name": "AgentExample2",
+                "path": "agents.agent_example2"
+            }
+        ]
     },
     "server": {
         "host": "0.0.0.0",
         "port": 5555,
-        "players_per_room": 2,
+        "nb_clients_per_room": 2,
+        "allow_multiple_connections": true,
+        "respawn_cooldown_seconds": 5.0,
+        "client_timeout_seconds": 2.0,
         "tick_rate": 60,
-        "game_duration_seconds": 300
+        "game_duration_seconds": 300,
+        "waiting_time_before_bots_seconds": 60,
```

Comment:
If you don't think students will modify a field, then don't include it here. The field will get the default value from the Config/ClientConfig/ServerConfig classes.